build-output/_multiple-bundles/readium-shared-js.js

```diff
@@ -11431,40 +11438,172 @@ var InternalLinksSupport = function(reader) {
 
     this.processLinkElements = function($iframe, spineItem) {
 
-        var epubContentDocument = $iframe[0].contentDocument;
+        var settings = reader.viewerSettings();
+        if (!settings || typeof settings.epubPopupFootnotes === "undefined")
+        {
+            //defaults
+            settings = new ViewerSettings({});
+        }
+        var epubPopupFootNotesAreEnabled = settings.epubPopupFootnotes;
+        if (epubPopupFootNotesAreEnabled) {
+            var doc = ( $iframe[0].contentWindow || $iframe[0].contentDocument ).document;
+            var $elements = $("*[epub\\:type]", doc);
+            if ($elements.length) {
+                
+                var sources = [];
+                var targets = [];
+                
+                $elements.each(function(i) {
+                    
+                    if ($elements[i].localName !== "a"
+                        && $elements[i].localName !== "aside") {
+                        return true; // continue
+                    }
+                    
+                    var $element = $($elements[i]);
+                    var epubType = $element.attr("epub:type"); 
 
-        $('a', epubContentDocument).click(function (clickEvent) {
-            // Check for both href and xlink:href attribute and get value
-            var href;
-            if (clickEvent.currentTarget.attributes["xlink:href"]) {
+                    if (epubType == "noteref" && $element[0].localName === "a") {
+                        sources.push($element);
+                    } else if ((epubType == "footnote" || epubType == "note" || epubType == "rearnote") && $element[0].localName === "aside") {
```

Comment:
The limitation to `aside` elements seems very artificial. Is this defined in the EPUB specification? Should the match be based solely on the `epub:type` value?
+ @JayPanoz

Reply:
This is a ` build-output` compiled file, not the raw source. Closing the comment.